src/components/toolbox/box/footerButton.js

```diff
@@ -0,0 +1,13 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './footerButton.css';
+
+const FooterButton = ({ children, ...rest }) => (
+  <button {...rest} className={styles.footerButton}>{children}</button>
```

Comment:
IMO, I would add the onClick as an explicit props here, to make it clearer.

Reply:
Done in https://github.com/LiskHQ/lisk-hub/pull/2338/commits/82a7d0cb5ad0702de7801f77154edae1106c568d